contracts/protocol/bases/mixins/FundsManager.sol

```diff
@@ -233,7 +233,9 @@ contract FundsManager {
                 // Copy the last token in the array to this index to fill the gap
                 tokenList[index] = tokenToMove;
                 // Reset index mapping. Should be index in tokenList array + 1
-                entityTokens[tokenToMove] = index + 1;
+                unchecked {
```

Comment:
Maybe we can do the `unchecked` also for lines 224-226, what do you think?

I mean, `tokenList` is supposted to have zero lenght only if the user has no available funds, but in that case we revert already in line 213, so `lastTokenIndex` should never overflow.